sdk/eventhub/azure-eventhub/samples/async_samples/send_and_receive_amqp_annotated_message_async.py

```diff
@@ -63,11 +68,11 @@ async def send_value_message(producer):
     print("Message of value body sent.")
 
 
-async def on_event(partition_context, event):
+async def on_event(partition_context: PartitionContext, event: Optional[EventData]) -> None:
     # Put your code here.
     # If the operation is i/o intensive, multi-thread will have better performance.
-    print("Received event from partition: {}".format(partition_context.partition_id))
-    raw_amqp_message = event.raw_amqp_message
+    print(f"Received event from partition: {partition_context.partition_id}")
+    raw_amqp_message = event.raw_amqp_message # type: ignore
```

Comment:
type ignore?

Reply:
I'm going to remove this. Since event is `Optional[EventData]` mypy was complaining that None doent have an attribute `raw_amqp_message`, we need to fix the typing for the `on_event` Callable as in my notes. If we are calling `on_event`, `Event Data` should'nt be `None`.